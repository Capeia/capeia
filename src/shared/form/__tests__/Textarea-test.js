// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<Textarea>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('../Textarea')
    const Component = require('../Textarea').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
