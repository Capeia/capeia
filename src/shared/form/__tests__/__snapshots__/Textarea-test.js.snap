// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Textarea> renders correctly 1`] = `
<textarea
  onInput={[Function]}
  style={
    Object {
      "height": "auto",
      "overflow": "hidden",
      "resize": "none",
    }
  }
/>
`;
